staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go

```diff
@@ -208,6 +208,8 @@ func (f *fancyResponseWriterDelegator) Flush() {
 }
 
 func (f *fancyResponseWriterDelegator) Hijack() (net.Conn, *bufio.ReadWriter, error) {
+	// fake a response status before protocol switch happens
+	f.processCode(http.StatusSwitchingProtocols)
```

Comment:
until we figure that out, please revert this change... setting 101 in non-switching cases is an API break